### PR TITLE
Refac casper

### PR DESCRIPTION
**Pull request by lawRathod**

### CODE REVIEW

File: libs/ledger-live-common/src/families/casper/bridge.integration.test.ts [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            amount: "1",
+            deploy: null,
+          }),
+
+          expectedStatus: {
+            errors: {
+              amount: new AmountRequired(
+                `Minimum CSPR to transfer is ${motesToCSPR(
+                  MINIMUM_VALID_AMOUNT
+                ).toNumber()} CSPR`
```
Comment (emmanuelm41) [inline libs/ledger-live-common/src/families/casper/bridge.integration.test.ts]: Could we convert this into some function and call it from wherever we need? So we avoid DRY.

**Review by emmanuelm41 — CHANGES_REQUESTED** [review]
Remember we need to switch to the frozen seed address on the integration tests, and update the snapshots!

File: libs/ledger-live-common/src/families/casper/bridge.integration.test.ts (hunk 1 above)
Comment (lawRathod) [reply libs/ledger-live-common/src/families/casper/bridge.integration.test.ts]: Yeah, about this. A function is perfect, should do it. I created a custom error in past called `InvalidMinimumAmount` but then figured that we will need to translate it to other languages, etc. just like other errors. 

Now if we use the error `AmountRequired` with custom message, the message doesn't appear in the UI. Which is not good... What do you think we should do? Does creating new error `InvalidMinimumAmount` make the most sense here?

Comment (lawRathod) [reply libs/ledger-live-common/src/families/casper/bridge.integration.test.ts]: Done added `InvalidMinimumAmount` custom error. Note that we do not provide transalations for this error class.

**Review by emmanuelm41 — APPROVED** [review]

**Pull request merged**